src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,9 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating(){
```

Comment:
should this have default implementation?

Reply:
Remember to add comments too. If the method is really simple, a line of comment would suffice